extension/src/paymentHandler/payment-utils.js

```diff
@@ -59,6 +59,19 @@ function createChangeTransactionStateAction(transactionId, transactionState) {
   }
 }
 
+function createChangeTransactionTimestampAction(
+  transactionId,
+  paymentObjTimestamp
```

Comment:
why do we always assume that it is the "payment object time stamp" and not just "timestamp" since it could be also a timestamp from the Adyen response, right?

Reply:
Undo the timestamp assignment in extension module.
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/9e35ab7f3f1285abb7119ef3a73d820293b3fba2